packages/@okta/vuepress-site/code/ios/index.md

```diff
@@ -35,7 +35,7 @@ This method works with both Okta Identity Engine and Okta Classic Engine.
 
 SDK: [Okta Mobile SDK for Swift](https://github.com/okta/okta-mobile-swift).
 
-Example: [Browser sign in](https://github.com/okta/samples-ios/tree/master/browser-sign-in).
+Example: [Browser sign-in](https://github.com/okta/samples-ios/tree/master/browser-sign-in).
```

Comment:
Actually, neither are right as we don't use sign-in as a noun. It would need to be Browser sign-in flow

Reply:
Changed to browser sign-in flow